src/main/java/core/basesyntax/Colour.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colour {

```

Comment:
Try to avoid mistakes in English words in your code. Colour -> Color

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/141337990/a285b16f-ccd8-46b9-ac83-bf81ef200cd3)

Reply:
Agree with you. I wrote my comment because in other cases you use American word like ColorSupplier. Try to be consistent